projects/lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angularx-social-login",
-  "version": "3.5.7",
```

Comment:
I think you can revert the changes in this file as well. Let me make the changes after testing and before publishing to npm.

Reply:
ok